examples/simple/main.tf

```diff
@@ -0,0 +1,24 @@
+provider "scaleway" {
+  region = local.region
+}
+
+locals {
+  name   = "ex-${basename(path.cwd)}"
+  region = "fr-par"
+  azs    = tolist(["fr-par-1", "fr-par-2"])
+  tags = [
+    local.name,
+    "terraform-scaleway-vpc-module"
+  ]
+}
+
+################################################################################
+# VPC Module
+################################################################################
+
+module "vpc" {
+  source = "../../"
```

Comment:
```suggestion
  source = "scaleway/vpc"
```

Reply:
Will rollback this. I got an error `The module address "scaleway/vpc" could not be resolved.
`